app/services/page.js

```diff
@@ -117,7 +117,7 @@ export default Service.extend({
     }
   }),
 
-  nextPage: computed('currentSection', 'currentPage', function() {
+  nextPage: computed('currentSection.pages', 'currentPage.url', function() {
```

Comment:
These are the only properties that are called on these objects. Technically, it was possible for this properties to change without the CP re-computing